### PR TITLE
Create Index Questions

### PR DESCRIPTION
**Pull request by nguyenvanson332000**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image
image

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,41 +1,38 @@
...
 
   def update
-
     if @user.update(user_params)
-      # Handle a successful update.
       flash[:success] = "Profile updated"
       redirect_to @user
     else
-      render 'edit'
+      render "edit"
```
Comment (rikai-tra) [inline app/controllers/users_controller.rb]:
```suggestion
      render :edit
```

File: .byebug_history [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+c
+@exam
+c
+n
+m
+n
+@question
+@exam
+c
+log_in(@user)
```
Comment (rikai-tra) [inline .byebug_history]: không đưa file này lên

File: app/controllers/admin/questions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+class Admin::QuestionsController < ApplicationController
+  before_action :load_exam, only: %i(create)
+  def new
+    @exam = Exam.find_by(id: params[:exam_id])
```
Comment (rikai-tra) [inline app/controllers/admin/questions_controller.rb]: thay thằng load_exam vào rồi thêm before_action thằng new

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (nguyenvanson332000) [reply app/controllers/users_controller.rb]: dạ cái này ở bên branch của bạn khác để em sửa lại ạ

File: .byebug_history (hunk 2 above)
Comment (nguyenvanson332000) [reply .byebug_history]: daj

File: app/controllers/admin/questions_controller.rb (hunk 3 above)
Comment (nguyenvanson332000) [reply app/controllers/admin/questions_controller.rb]: à dạ

File: app/views/admin/questions/_question.slim [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.center.col-lg-12.view-question
+  .create-question id="question-form-#{@exam.id}"
+  ul.question-ul
+    - @exam.questions.each do |question|
```
Comment (rikai-hai) [inline app/views/admin/questions/_question.slim]: Không query ở view và helper

**Review by rikai-hai — APPROVED** [review]

**Pull request merged**